cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -21,11 +22,13 @@ OPTIONS
            definition. If a package is not specified, the output will be written
            to a 'temp' directory in the current location.
 
-       --mode client | service | proxy
+       --mode client | service
```

Comment:
```suggestion
       --mode mode
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/c041ce5fcdfe872fc4378580d5d0b975ecbde922